include/rocksdb/cache.h

```diff
@@ -48,6 +48,8 @@ enum CacheMetadataChargePolicy {
 const CacheMetadataChargePolicy kDefaultCacheMetadataChargePolicy =
     kFullChargeCacheMetadata;
 
+const double kDefaultLowPriortyPoolRatio = -1.0;
+
```

Comment:
Defining a namespace-scope variable in the header like this can lead to ODR violations. Also, the variable name is misspelled (then again, the simplest solution would be to ditch this constant)